src/ui/liferea_shell.c

```diff
@@ -908,8 +908,8 @@ static const GtkActionEntry liferea_shell_action_entries[] = {
 	 G_CALLBACK(on_menu_update_all)},
 	{"MarkAllFeedsAsRead", "gtk-apply", N_("Mark All As _Read"), NULL, N_("Marks read every item of every subscription."),
 	 G_CALLBACK(on_menu_allfeedsread)},
-	{"ImportFeedList", "gtk-open", N_("_Import Feed List..."), NULL, N_("Imports an OPML feed list."), G_CALLBACK(on_menu_import)},
-	{"ExportFeedList", "gtk-save-as", N_("_Export Feed List..."), NULL, N_("Exports the feed list as OPML."), G_CALLBACK(on_menu_export)},
+	{"ImportFeedList", "gtk-open", N_("_Import Feed List…"), NULL, N_("Imports an OPML feed list."), G_CALLBACK(on_menu_import)},
+	{"ExportFeedList", "gtk-save-as", N_("_Export Feed List…"), NULL, N_("Exports the feed list as OPML."), G_CALLBACK(on_menu_export)},
```

Comment:
Ok, I've force-pushed a new commit with this change reverted.

Still, I wonder why the translation breaks if changing the application in a way the compiler suggests… Seems like Unicode is still a brand-new technology for tools such as compilers or gettext or whatever is breaking because of the triple-dot.